TSC-Charter.md

```diff
@@ -81,21 +81,31 @@ Subject to such policies as may be set by the Board, the TSC is
 responsible for all technical development within the Node.js Foundation,
 including:
 
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and any coding standards.
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Maintaining the list of additional Collaborators
+* Development process and any coding standards
 * Mediating technical conflicts between Collaborators or Foundation
-projects.
+projects
+* Management of the `nodejs-private` GitHub organization
 
 The TSC will define Node.js Foundation’s release vehicles and serve as
 Node.js Foundation’s primary technical liaison body with external open
 source projects, consortiums and groups.
 
+The TSC shares responsibility with the Node.js Community Committee ("CommComm")
+for:
+
+* Management of the `nodejs` GitHub organization
```

Comment:
I assume **GitHub repository hosting** is replaced by  **Management of the `nodejs` GitHub organization**

Right?

Reply:
Yes. "GitHub repository hosting" is not what TSC was ever responsible for in the first place. It would have been more accurate to say "GitHub repository management" but I think "Management of the nodejs GitHub organization" is a superset of that.

Reply:
One thing for TSC to consider: Approving apps. TSC has traditionally been more conservative about wanting/approving apps than CommComm. This may no longer be A Thing because of `nodejs-private`, but maybe it is?